app/controllers/admins_controller.rb

```diff
@@ -0,0 +1,17 @@
+class AdminsController < ApplicationController
+  def category
+    @categories = Category.all
+  end
+
+  def word
+    @words = Word.all
+  end
+
+  def word_answer
+    @word_answers = WordAnswer.all
+  end
+
+  def user
+    @users = User.all
+  end
```

Comment:
@Masa08 Each of these methods should be in their own controller. 

Example `def category` should be `def index` inside `CategoriesController`.